commitlint/plugins.test.ts

```diff
@@ -476,6 +476,19 @@ test("footer-references-existence4", () => {
     expect(footerReferenceExistence4.status).toBe(0);
 });
 
+test("footer-references-with-EOL", () => {
+    let commmitMsgwithEOLFooter =
+        "foo: this is only a title" +
+        "\n\n" +
+        "Bla bla blah[1]." +
+        "\n\n" +
+        "[1]\nhttp://foo.bar/baz";
+    let footerReferenceWithEOL = runCommitLintOnMsg(commmitMsgwithEOLFooter);
+    expect(footerReferenceWithEOL.output[1].toString().includes("EOL")).toBe(
```

Comment:
This expectation looks kinda weird

Reply:
Isn't a better way to "expect" this?

Reply:
What if we change the message later?

Reply:
That was the best way I could think of. We are sure that there is an error we just want to check if the message is what we expect. Do you have any suggestions?

Reply:
> That was the best way I could think of. We are sure that there is an error we just want to check if the message is what we expect. Do you have any suggestions?

true, error always happens here so not really, I don't have a suggestion, I guess this is our only option then.

Reply:
I agree that it's not ideal and EOL was the shortest thing we could think of that may never change in the error message :sweat_smile: